backend/app/workers/register_organization_with_contacts_on_crm_worker.rb

```diff
@@ -0,0 +1,144 @@
+class RegisterOrganizationWithContactsOnCrmWorker < ApplicationWorker
+  attr_reader :enrollment
+
+  def perform(enrollment_id)
+    @enrollment = Enrollment.find_by(id: enrollment_id)
+
+    return unless enrollment
+
+    crm_company = find_or_create_company_on_crm
+
+    valid_team_members.each do |team_member|
+      add_contact_to_company(
+        find_or_create_contact_on_crm(team_member),
+        crm_company
+      )
+    end
+  end
+
+  private
+
+  def create_contact_on_crm(team_member)
+    crm_client.create_contact(
+      email: team_member.email,
+      firstname: team_member.given_name,
+      lastname: team_member.family_name,
+      phone: team_member.phone_number,
+      type_de_contact: humanize_contact_type(team_member.type)
+    )
+  end
+
+  def create_company_on_crm
+    crm_client.create_company(
+      siret: organization.siret,
+      name: organization.insee_payload["nom_raison_sociale"],
+      categorie_juridique: organization.insee_payload["categorie_juridique"],
+      n_datapass: enrollment.id.to_s
```

Comment:
Ici faut ajouter: 
- type d'entité : Fournisseur de service partout
- bouquet: API Part/Ent/FC
- solution logicielle : démarche/formulaire pré-rempli
- Intégrateur : Qui implémentera l’API ?
  - Votre équipe de développeur
  - Label de l'entreprise (on se tape sur siret)
  - Autre : peut-on faire des create_or_update dans Hubspot ? (oui) si oui/pas trop relou, ajouter sous la forme Autre: INPUT

Reply:
vu IRL on peut merger en l'état on itérera pour ajouter ça ensuite